src/components/subsection/subsection.jsx

```diff
@@ -11,11 +11,14 @@ import styles from './subsection-styles';
 class Subsection extends React.Component {
   constructor(props) {
     super(props);
-    this.theme = createTheme();
 
     this.state = {
       toggled: this.props.toggled,
     };
+
+    const matchMedia = props.matchMedia;
+    const breakpoints = createBreakpoints();
+    this.onDesktop = matchMedia(`(min-width: ${breakpoints.md}px)`).matches;
```

Comment:
```

Reply:
the problem with importing local to component theme related stuff is that it becomes impossible to override it via ThemeProvider in the apps